Cython/Shadow.pyi

```diff
@@ -76,20 +78,16 @@ class PointerType(CythonType, Generic[_T]):
 class ArrayType(PointerType[_T]):
     def __init__(self) -> None: ...
 
-#class StructType(CythonType, Generic[_T]):
-#    def __init__(
-#        self,
-#        value: List[Type[_T]] = ...
-#    ) -> None: ...
-
 def index_type(
     base_type: _T, item: Union[tuple, slice, int]) -> _ArrayType[_T]: ...
 
 def pointer(basetype: _T) -> Type[PointerType[_T]]: ...
 
 def array(basetype: _T, n: int) -> Type[ArrayType[_T]]: ...
 
-#def struct(basetype: _T) -> Type[StructType[_T]]: ...
+def struct(**members: type) -> Type[Any]: ...
+
+def union(**members: type) -> Type[Any]: ...
```

Comment:
`type`?

Reply:
No, there's a difference afaik. See https://docs.python.org/3/library/typing.html#typing.Type

Reply:
sorry for the misunderstanding, I meant this `type`: `**members: type`
I don't understand what that should mean. This is probably a typo?

Reply:
Ah good catch, I think that should be `Type` as well. I'll double-check tomorrow.

Reply:
Type is deprecated as of 3.9, since the builtin now supports indexing. But since this is a stub file you can use it in any version.

Reply:
Actually it'd be better to use `type` exclusively, to future-proof since eventually `typing.Type` will be removed, similarly `List` and `Dict` can be swapped over and their imports removed.

Reply:
I think what @0dminnimda was asking is how a dict can be of type `type`. That's a special case for `**kwargs` dicts, where only the type of the dict values is mentioned (because they keys are always strings by design):

https://peps.python.org/pep-0484/#arbitrary-argument-lists-and-default-argument-values